src/electron/views/screenshot/screenshot-view.tsx

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { NamedFC } from 'common/react/named-fc';
+import * as React from 'react';
+
+export type ScreenshotViewProps = {};
+export const ScreenshotView = NamedFC<ScreenshotViewProps>('ScreenshotView', props => {
+    return (
+        <div>
+            <h1 tabIndex={0}>This is the screenshot view</h1>
```

Comment:
aren't we at a point were we can actually render the `<Screenshot>` component? or is that outside this PR's scope?

Reply:
I think this is outside of the scope of this PR

Reply:
Yep, it's covered later in 1628929